core/DataTable/Row.php

```diff
@@ -361,6 +395,22 @@ public function setColumn($name, $value)
         $this[$name] = $value;
     }
 
+    /**
+     * @param mixed $value
+     */
+    public function setFormattedColumn(string $name, $value): void
```

Comment:
Could perhaps return type to `self` and `return $this` to support fluent methods chaining?

Reply:
Generally I like the idea of method chaining. But we imho should decide to either add that to all methods in a class or to none. Everything between might confuse more than it helps in the end.